src/microbe_stage/EnergyBalanceInfoEvaluated.cs

```diff
@@ -0,0 +1,64 @@
+﻿/// <summary>
+///   Info regarding a microbe's energy balance in a patch, excluding consumption and production
+/// </summary>
+/// <remarks>
+///   <para>
+///     This does not take special modes the microbe can be into account, for example engulfing or binding
+///   </para>
+///   <para>
+///     The consumption and production is not tracked, use <see cref="EnergyBalanceInfoFull"/> to store it
+///   </para>
+/// </remarks>
+public class EnergyBalanceInfoEvaluated
```

Comment:
I don't really get why the name of this is `Evaluated` I'd name this like "simple" or "base" (if that other info is made to inherit from this).

Reply:
Since it contains the final evaluated balance rather than a dictionary of the contribution from each organelle.

Reply:
But where is the non-evaluated variant? That's what's confusing me is that if that doesn't actually exist, then it is just confusing to specify that this class is "evaluated" when there isn't a non-evaluated variant.

Reply:
I have renamed it for your benefit.